src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +81,80 @@ Common questions and requests include:
-    - The Hero leaves a comment saying the ticket is being triaged
-2. Each writer should spend 10-15 minutes/day triaging their assigned tickets 
-    - Writer either rewrites or closes the ticket
-      - If the writer closes the ticket, they must leave a comment explaining why.
-    - Writer needs to add: 
-      - Acceptance criteria
-      - User story
-      - Action items
-      - Resources 
-    - While rewriting, focus on making the ticket as swarmable as possible
-    - Assign a priority [See priority info in our team's `DOC Jira and Github fields - Project 401 - 2022` doc]
-    - Add the following label: `doc_groomed`
-3. Triaged tickets enter sprints in one of two ways:
-    - **Scrum leaders**: Bring in tickets based on marked priority
-    - **Liaisons**: Bring in high value issues
-    - **Squads**: During backlog grooming the team can chat about bringing in tickets to fill out a sprint
-
 <Callout variant="important">
 For writers in Europe:
 
 - Route incoming issues to the liaison as you have time. As writers in Europe are "eternal heroes" it isn’t fair to have you do this every day.
 - You should still triage issues routed to you from your liaisonships once per day.
  </Callout>
 
+## Customer feedback sidekick [#cf_sidekick]
+
+Customer feedback from our docs site creates tickets in Jira. It's the sidekick's responsibility to look at every ticket that comes in (usually about 5-10 per day) and make decisions about each one.
+
+We have three options for these tickets:
+
+1. Close it.
+2. Fix it.
+3. Backlog it.
+
+<Callout variant="important">
+Unlike the hero, we don't take any points out of our sprint for the sidekick work. If you're spending more than 60 minutes per day on customer feedback tickets, that's too much time! Please let your manager know.
+ </Callout>
+
+### Close customer feedback
+
+Most customer feedback that comes in isn't actionable, either because it verges on nonsense or because the feedback is too non-specific to act on.
+
+For example:
+
+`These docs are bad.`
+
+or 
+
+`I can't log in to Netflix.`
+
+Follow this process to close these:
+
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+2. Set the Workflow status to **Closed** 
+3. Set the Resolution to **Cannot Reproduce**, **Filed by Mistake**, or **Won't Do** depending on what feels most appropriate.
+4. Add a brief note about why you closed it. Usually, "Not actionable" is plenty.
+
+### Fix customer feedback
+
+Sometimes, though rarely, a customer feedback ticket comes in that's highly actionable and is a quick fix, such as a broken link or an outdated image. 
+
+Follow this process to close these:
+
+1. Create a PR to address the issue.
+2. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+3. Set the Workflow status to **Closed** 
+4. Set the Resolution to **Done**.
+5. Add a link to the PR.
+6. Work the PR to its merge at your leisure.
```

Comment:
I don't think we should close the ticket until the PR is merged, if it's actively being worked. I think that encourages hidden work in a way I don't love.

Reply:
@austin-schaefer Strong disagree on this one. How many PR's does our team open and then not close? My biggest worry about the customer feedback tickets is that they become this gloomy cloud hanging over everyone. In my view, if a PR is opened, the customer feedback ticket should be closed. 

We can easily link from the customer feedback ticket to the PR, but we can't go the other way, because of security concerns. It adds more toil than it's worth, in my view, to add that unnecessary delay to closing these customer feedback tickets. I think we need to stop treating and thinking about customer feedback tickets as special, holy objects that we treat with veneration.

Reply:
I like your gloomy cloud analogy, but for me the gloomy cloud is there even if the ticket is nominally closed. If someone is doing shadow-work on a PR, or letting a PR languish, on a nominally closed Jira, then we've simply reduced our ability to help/see what's going on. If the CF is still open and, say, sitting in on-deck, we get an automatic forcing function to ensure the related work gets chased down or cancelled.

Reply:
I've updated the procedure so that we're only closing Jira tickets once the PR is merged in.